website/files/models/base.py

```diff
@@ -573,10 +586,10 @@ def touch(self, auth_header, revision=None, **kwargs):
             self.generate_waterbutler_url(revision=revision, meta=True, **kwargs),
             headers=headers,
         )
+        # Returns None to addon_view_or_download_file 'version'
```

Comment:
Having a bit of trouble understanding this comment - seems like the ```File.touch``` method is used elsewhere than to just return to the specific view -- I could be wrong though!